website/docs/tutorial-nlp.md

```diff
@@ -0,0 +1,174 @@
+
+```python
+>>> # Calculate the POS tags.
+>>> df["pos tag"] = df["powers_text"].pipe(hero.pos_tag)
+>>> df.head(3)[["name", "powers_text", "pos tag"]]
+```
+![](/img/tutorial-nlp-S6.png)
+
+Just like with the noun chunks, we now extract the adjectives by writing a function that extracts them from a list of POS-tags and applying that function to the whole column.
+
+```python
+def adjectives_from_pos_tags(list_of_pos_tags):
+    # Return a list of all words whose part-of-speech is "ADJ", so all adjectives.
+    return [word for (word, kind, _, _, _) in list_of_pos_tags if kind == "ADJ"]
+```
+
+Again, just apply:
+
+```python
+>>> df["powers"] = df["pos tag"].apply(adjectives_from_pos_tags)
+>>> # Look at the interesting rows we defined above again.
+>>> df[interesting_rows][["name", "pos tag", "powers"]]
+```
+![](/img/tutorial-nlp-S7.png)
+
+
+## Named Entities - Where do our superheroes live?
+
+Having found out so much about our superheroes, we're now interested in where they live. To find that out, we use `hero.named_entities` to find each history text's _Named Entities_. Those are exactly what the name suggests - the entities, e.g. "Yesterday" (a date), "New York" (a location), "Dracula" (a person). We're interested in locations. Those get the tag "GPE" (geographical entity). Thus, we'll first use `named_entities` to get a list of named entities for each row, and then apply a function to extract the most-mentioned geographical entity from the named entities.
+
+```python
+>>> # Calculate the Named Entities.
+>>> df["named entities"] = df["history_text"].pipe(hero.named_entities)
+>>> df.head(3)[["name", "history_text", "named entities"]]
+```
+![](/img/tutorial-nlp-S8.png)
+
+Here's the function to extract the most common geographical entity:
+
+```python
+def location_from_named_entities(list_of_named_entities):
+    # Collect all geographical entities.
+    mentioned_locations = [
+        entity for (entity, label, _, _) in list_of_named_entities if label == "GPE"
+    ]
+    # If any were found, return the most common one.
+    if mentioned_locations:
+        most_frequently_mentioned_location = max(
+            mentioned_locations,
+            key=mentioned_locations.count
+        )
+        return most_frequently_mentioned_location
+    else:
+        return ""
+```
+
+Let's apply the function and take a look at a few results.
+
+```python
+>>> df["location"] = df["named entities"].apply(location_from_named_entities)
+>>> df[["name", "location", "named entities"]].head(5)
+```
+![](/img/tutorial-nlp-S9.png)
+
+We get some good and some not-so-good results. There's certainly a lot more fun that can be had with this dataset!
+
+## Recap
+
+In this tutorial, we took a look at all of Texthero's core NLP functions (which are always being expanded and improved). Hopefully you've learned that
```

Comment:
The last part sounds like, that the user should have learned, that Texthero is constantly improving 🥇  Maybe a ':' or new line would help

Reply:
:octopus: :british_indian_ocean_territory: